packages/core/src/extensions/SuggestionMenu/getDefaultEmojiPickerItems.ts

```diff
@@ -9,13 +9,32 @@ import {
 } from "../../schema/index.js";
 import { DefaultGridSuggestionItem } from "./DefaultGridSuggestionItem.js";
 
-let data:
-  | Promise<{
-      default: EmojiMartData;
-    }>
-  | undefined;
-
+// Temporary fix for https://github.com/missive/emoji-mart/pull/929
 let emojiMart: typeof import("emoji-mart") | undefined;
+let emojiData: EmojiMartData | undefined;
+
+async function loadEmojiMart() {
+  if (!emojiMart || !emojiData) {
+    // load dynamically because emoji-mart doesn't specify type: module and breaks in nodejs
+    const [emojiMartModule, emojiDataModule] = await Promise.all([
+      import("emoji-mart"),
+      // use a dynamic import to encourage bundle-splitting
+      // and a smaller initial client bundle size
+      import("@emoji-mart/data"),
+    ]);
+
+    emojiMart =
+      "default" in emojiMartModule ? emojiMartModule.default : emojiMartModule;
+    emojiData =
+      "default" in emojiDataModule
+        ? (emojiDataModule.default as EmojiMartData)
+        : (emojiDataModule as EmojiMartData);
+
+    emojiMart.init({ data: emojiData });
```

Comment:
I think this needs an `await`

Reply:
Hm, wasn't clear that it was async in the docs https://github.com/missive/emoji-mart?tab=readme-ov-file#-emoji-component , but won't hurt to await